easybuild/easyconfigs/o/OpenBUGS/OpenBUGS-3.2.3-GCC-8.3.0.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Author: Guilherme Balzana
+# Research Computing Services - University of Cambridge
+
+easyblock = 'ConfigureMake'
+
+name = 'OpenBUGS'
+version = '3.2.3'
+
+homepage = 'http://openbugs.net'
+description = """BUGS is a software package for performing Bayesian inference
+    Using Gibbs Sampling. The user specifies a statistical model, of (almost)
+    arbitrary complexity, by simply stating the relationships between related
+    variables. The software includes an ‘expert system’, which determines an
+    appropriate MCMC (Markov chain Monte Carlo) scheme (based on the Gibbs sampler)
+    for analysing the specified model. The user then controls the execution of the
+    scheme and is free to choose from a wide range of output types.
+"""
```

Comment:
I suspect these spaces might end up inside the description int he module file, so that might look ugly